halconfig/echo.yml

```diff
@@ -21,8 +21,6 @@ orca:
   enabled: ${services.orca.enabled:false}
   baseUrl: ${services.orca.baseUrl:http://localhost:8083}
 
-endpoints.health.sensitive: false
```

Comment:
What was this removed?  It seems unrelated to the current change.

Reply:
I was having issues starting Echo, and the logs spit out this error message for me:

```
The use of configuration keys that are no longer supported was found in the environment:

Property source 'applicationConfig: [file:/usr/local/google/home/ttomsu/.spinnaker/echo.yml]':
        Key: endpoints.health.sensitive
                Reason: Endpoint sensitive flag is no longer customizable as Spring Boot no longer provides a customizable security auto-configuration .
```
Turns out my startup error was something else, but removing this line made this error go away.